src/query/service/tests/it/interpreters/union.rs

```diff
@@ -127,32 +127,72 @@ fn get_create_table_sql(table_name: &str, ty: &str, nullable: bool) -> String {
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
 async fn test_union_output_type() -> Result<()> {
+    fn null_str(nullable: bool) -> &'static str {
+        if nullable { "null" } else { "not_null" }
+    }
+
+    let fixture = TestFixture::new().await;
+
+    // Permutation of all data types: ALL_CREATABLE_TYPES x [true, false]
+    let mut data_types = vec![];
+
+    // Prepare all tables
     for ty1 in ALL_CREATABLE_TYPES {
         for nullable1 in [true, false] {
-            let table1_sql = get_create_table_sql("t1", ty1, nullable1);
+            let type_str = format!("{}_{}", ty1, null_str(nullable1));
+
+            let t1_name = format!("t1_{}", type_str);
+            let table1_sql = get_create_table_sql(&t1_name, ty1, nullable1);
```

Comment:
we should modify `get_create_table_sql` to create all types in one table.

Reply:
Next PR:)